src/bundle/Command/CronRunCommand.php

```diff
@@ -46,6 +47,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $cron->setExecutor(new Executor());
         $cron->setResolver($resolver);
 
-        $cron->run();
+        $reports = $cron->run();
+
+        while ($cron->isRunning()) {}
+
+        if($this->getContainer()->has('monolog.logger.cronjob')){
+            $logger = $this->getContainer()->get('monolog.logger.cronjob');
+            if($logger){
+                foreach ($reports->getReports() as $report) {
+                    $extraInfo = array(
+                        'command' => $report->getJob()->getProcess()->getCommandLine(),
+                        'exitCode' => $report->getJob()->getProcess()->getExitCode()
+                    );
+                    foreach($report->getOutput() as $reportOutput){
+                        $logger->info($reportOutput, $extraInfo);
+                    }
+                    if(!$report->isSuccessful()){
+                        foreach($report->getError() as $reportError){
+                            if(!empty(trim($reportError))){
+                                $logger->error(trim($reportError), $extraInfo);
```

Comment:
Do not repeatedly `trim` the very same string. Use local variable. (Same goes for `$report->getJob()->getProcess()`)

Reply:
@Nattfarinn I fixed it